quattor/types/hardware.pan

```diff
@@ -20,9 +20,12 @@ type structure_ram = {
 }
 type structure_cpu = {
     include structure_annotation
-    "speed"  : long # "CPU clock speed in MHz"
-    # Number of cores on each CPU chip, defaults to 1.
+    @{ desc = CPU clock speed in MHz }
+    "speed"  : long
+    @{ desc = Number of cores on each CPU chip, defaults to 1. }
```

Comment:
I think "defaults to 1", fails the "obvious-ness" test of a good comment and should be removed.